luaui/Widgets/gui_options.lua

```diff
@@ -2926,6 +2926,13 @@ function init()
 			  end
 		  end,
 		},
+		{ id = "gridmenu_alwaysreturn", group = "control", category = types.advanced, name = Spring.I18N('ui.settings.option.gridmenu_alwaysreturn'), type = "bool", value = (WG['buildmenu'] ~= nil and WG['buildmenu'].getShowPrice ~= nil and WG['buildmenu'].getShowPrice()), description = Spring.I18N('ui.settings.option.gridmenu_alwaysreturn_descr'),
```

Comment:
Appreciate a way to make this option visibility depend on gridmenu active

Reply:
made it so.